app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,25 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @current_post = Post.find(params[:post_id])
+    @like = @current_post.likes.new(like_params)
+    @like.post_id = @current_post.id
+    @like.author_id = current_user.id
+    if @like.save
+      # flash[:success] = 'New like saved successfully'
+      # redirect_to user_posts_path(current_user, @current_post)
```

Comment:
- [OPTIONAL] Same here, please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮